zer0-os/zOS
# feat: add zApp Staking to app container

### What does this do?

- Adds a new external package: [`@zero-tech/zapp-staking`](https://www.npmjs.com/package/@zero-tech/zapp-staking) - this is the Staking zApp.
- Adds the `App.Staking` enum value to the list of active apps in `lib/apps/index.ts`.
- Renders Staking zApp when `selectedApp === App.Staking`.

Short answer: it adds zApp Staking to the list of apps.

### Why are we making this change?

To break ground on the relationship between zApps and zOS - this PR will act as validation of our process.

### How do I test this?

- Click "Staking" in the app selector.
- Check that the "Staking" app renders.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  3. How will this affect performance?
  4. Does this change any APIs?
